src/Admin/Settings.php

```diff
@@ -153,6 +153,20 @@ public function register_settings( $settings ) {
 						'type'    => 'text',
 						'default' => '',
 					],
+					[
+						'name'    => esc_html__( 'Client Id', 'instamojo-for-give' ),
+						'desc'    => esc_html__( 'Please enter the client id from your LIVE Instamojo Account.', 'instamojo-for-give' ),
+						'id'      => 'instamojo_get_client_id',
```

Comment:
@Ankur0812 rename the `id` to use `mg_instamojo_for_givewp_client_id` and similar for client secret `mg_instamojo_for_givewp_client_secret`